### ISSUE TITLE
Use other security categorization frameworks besides FIPS 199 in an SSP

### ISSUE THREAD

**Issue by RS-Credentive** [usnistgov/OSCAL#1795]
### User Story

As a system owner, I should be able to define the security characteristics of my system in frameworks other than the FIPS-199. For example, suppose I am operating a Certificate Authority under the Federal PKI. In that case, I will identify the Certificate Policies contained in the certificates that my CA will issue, identifying the controls that apply to my system.

### Goals

I would like to extend my system security plan's Security Sensitivity Level and Security Impact Level to reference other security frameworks or define a new child element of the System-Characteristics element to define an alternative categorization model.

### Acceptance Criteria

- [ ] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [ ] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [ ] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.
- [ ] the OSCAL validation tool can successfully validate an SSP that doesn't contain fips-199 impact levels under system characteristics

**Comment by aj-stein-nist** [comment]
During sprint review, we discussed as a team the shorter-term fix (not a backwards-breaking change) and the longer-term (potentially a backwards-breaking change) that is more elaborate. We decided and recommended to move forwarded with the shorter-term, low-impact change to relax the allowed-values constraint and not merging a more flexible system for custom categorization models. We will move this forward to Sprint 74 and I will pick up with @nikitawootten-nist when he is available.
